code/style.css

```diff
@@ -1,4 +1,229 @@
+  padding: 0; 
+}
+.site-name span{
+  color: #2c6e49;
+}
+/* header menue row styling*/
+.header-row2{
+  display: none;
+}
+
+/* site content styling*/
+.content{
+  background-color: #d8e2dc;
+ }
+.row{
+  display: flex;
+  flex-direction: column;
+  align-items:stretch;
+}
+.top-fig{
+  display: block;
+  position:relative 
+} 
+.top-fig p{
+  display: none;
+  color:#fff;
+  background-color:#c9c9c9;
+  width: 100%;
+  padding: 10px;
+}
+.top-fig:hover p{
+  position:absolute; top:75%; left:0;
+  display: block;
+}
+.top-fig figcaption { 
+  position:absolute; Top:0; left:0; 
+  padding: 10px; 
+ }
+
+.weekly-rec{
+  display: grid;
+  grid-template-columns: repeat( auto-fit, minmax(250px, 1fr) );
+  grid-gap: 1rem;
+  margin: 20px;
+  padding:0;
+}
+.featured {
+  background-color: white;
+}
+.rec-img {
+  width: 100%;
+  
+}
+.weekly-img{
+  margin: 0;
+  padding: 0;
+  position:relative;
+}
+.weekly-img figcaption{
+  opacity: 0;
+  position:absolute; Top:75%; left:0; 
+  padding: 10px; 
+  background-color: #c9c9c9;
+  font-size:1rem ;
   color: #fff;
+  width: 100%;
+  display: flex;
+  justify-content:space-evenly
+}
+.featured:hover figcaption{
+  opacity: 1;
+}
+.rec-link{
+  text-decoration: none;
+  color: #333;
+}
+.discription{
+ font-weight:bold;
+ text-align: center;
+}
+.other{
+  display: flex;
+  flex-direction: column;
+  margin: 20px 0 0 20px;
+  padding: 0 20px 0 20px;
+}
+.nav-list{
+ list-style:none;
+ padding: 0;
+ margin: 0;
+}
+.nav-rec{
+  width: 100%;
+  align-items: center;
+  padding: 20px 0;
+  margin-bottom: 10px;
+  border-bottom: 0.5px solid #333;
+}
+
+/*footer styling*/
+ .footer{
+   display: flex;
+   background-color: #2c6e49;
+   justify-content: center;
+   padding: 0;
+   margin: 0;
+ }
+
+.fa-facebook, .fa-twitter, .fa-instagram {
+  color: white;
+  padding: 20px;
+  font-size: 30px;
+  width: 50px;
+  text-align: center;
+  text-decoration: none;
 }
+
+  /* media queries */
+
+@media (min-width: 992px) {
+  .header{
+    display:flex;
+    flex-direction: column;
+  }
+
+  .logo{
+    display: inline;
+  }
+  .menu-bars{
+    display: none;
+  }
+  .site-name {
+    font-size: 3rem;
+  }
+  .header-row2{
+    display: block;
+  }
+  .nav{
+    padding: 0;
+    margin: 0 ;
+    padding: 0 80px;
+    justify-content: center;
+  }
+  .options{
+    list-style-type: none ;
+    display: flex;
+    flex: 1;
+    padding: 0 60px 0 0; 
+    font-size: 1rem;
+    width: 100%;
+  }
+  .link {
+    text-decoration: none;
+    color: rgb(8, 8, 8);
+  }
+  .link:hover{
+    background-color: darkgrey;
+  }
+  /*animation*/
+
+  .top-fig figcaption { 
+    animation-duration: 3s;
+    animation-name: slidein;
+  }
+  @keyframes slidein {
+    from {
+      margin-left: 100%;
+      width: 300%;
+    }
+    50% {
+      
+      font-size: 300%;
+      margin-left: 25%;
+      width: 150%;
+      position:absolute; Top:150px; left:0;
+    }
+    to {
+      margin-left: 0%;
+      width: 100%;
+    }
```

Comment:
Ah, I was wondering how you managed to do the slide-ins. I see how it works here. Nice!